projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-shape.php

```diff
@@ -167,15 +380,28 @@ public function is_possible() {
 	}
 }
 
+/**
+ * Jetpack tiled gallery three columns class.
+ */
 class Jetpack_Tiled_Gallery_Three_Columns extends Jetpack_Tiled_Gallery_Shape {
+	/**
+	 * The shape.
+	 *
+	 * @var array
+	 */
 	public $shape = array();
 
+	/**
+	 * Constructor class.
+	 *
+	 * @param object $images - the images.
+	 */
 	public function __construct( $images ) {
 		parent::__construct( $images );
 
 		$total_ratio              = $this->sum_ratios( $this->images_left );
 		$approximate_column_ratio = $total_ratio / 3;
-		$column_one_images        = $column_two_images = $column_three_images = $sum = 0;
+		$column_one_images        = $column_two_images = $column_three_images = $sum = 0; // phpcs:ignore Squiz.PHP.DisallowMultipleAssignments.Found
```

Comment:
Why not just fix it?
```suggestion
		$column_one_images        = 0;
		$column_two_images        = 0;
		$column_three_images      = 0;
		$sum                      = 0;
```